.devcontainer/postcreatecommand.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+git config --global --add safe.directory "$PWD"
```

Comment:
`pre-commit install` fails without this. The error log says to add this line. 
1. Is this only needed on Windows?
2. Does it cause problems on non-Windows environments?